user-guide/Advanced_Functionality/Databases/OpenSearch_database.md

```diff
@@ -16,23 +16,297 @@ Supported versions:
+> The rootCA.crt can also be installed on a Windows machine by double-clicking it in case you want to query the OpenSearch from a different computer. As *Store Location* when adding the certificate, use *Local Machine* and place the certificate in *Trusted Root Certification Authorities*.
+> It is the same procedure like when you would use Elastic, [Client-server TLS encryption](https://docs.dataminer.services/user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.html#client-server-tls-encryption). So once the certificate is imported, there is trust between your machine running DataMiner and the OpenSearch database/cluster.
+
+###### Remove the demo-certificates
+
+Remove the demo-certificates located in /etc/opensearch:
+
+```bash
+cd /etc/opensearch
+```
+
+```bash
+sudo rm -f *pem
+```
+
+###### Update the trusted root certificates with rootCA.crt
+
+Navigate to the location where your rootCA.crt file is stored and copy it using the following commands:
+
+```bash
+sudo cp rootCA.crt /usr/local/share/ca-certificates/
+```
+
+Next update the trusted root certificates:
+
+```bash
+sudo update-ca-certificates
+```
+
+You can verify the rootCA.crt using the following command, it should return the status OK.
+
+```bash
+openssl verify rootCA.crt
+```
+
+###### Configure users
+
+Generate a new hash for the admin user using [Configure a user](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/debian/#configure-a-user) section in the OpenSearch help. As the help indicated, remove all demo users except the *admin* user an replace the hash for admin with the generated hash.
+
+##### Setup TLS in the OpenSearch.yml file
+
+```yml
+plugins.security.disabled: false
+
+#Transport layer TLS
+plugins.security.ssl.transport.keystore_type: PKCS12
+plugins.security.ssl.transport.keystore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.transport.keystore_password: ReplaceMeByGeneratedPasswordByGithubScript
+plugins.security.ssl.transport.truststore_type: PKCS12
+plugins.security.ssl.transport.truststore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.transport.truststore_password: ReplaceMeByGeneratedPasswordByGithubScript
+
+#REST Layer TLS
+plugins.security.ssl.http.enabled: true
+plugins.security.ssl.http.keystore_type: PKCS12
+plugins.security.ssl.http.keystore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.http.keystore_password: ReplaceMeByGeneratedPasswordByGithubScript
+plugins.security.ssl.http.truststore_type: PKCS12
+plugins.security.ssl.http.truststore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.http.truststore_password: ReplaceMeByGeneratedPasswordByGithubScript
+
+plugins.security.allow_default_init_securityindex: true
+plugins.security.nodes_dn:
+  - 'CN=FQDNOpenSearchNode1,OU=DMS,O=OpenSearch,C=BE'
```

Comment:
Probably better to replace DMS in OU by YourClusterName to make sure they use the correct clustername.

Reply:
Fixed in commit from 02/06/2023